From 158df3cbfe0a934487f741fb1f953ef17fb0d98a Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Mon, 18 Nov 2024 15:32:47 +0200
Subject: [PATCH] [SW-207386] gaudi3: avoid telemetry config write

According to spec FW should not write the config register, this register
is written by host only.

Change-Id: Ic354fecc137a47dd16667173609b2f0fca9d32a5
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index be31b03..0db3a90 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -392,10 +392,8 @@
 	rst_src = hl_rst_src_get();
 	if ((rst_src != HL_COLD_RST) && (rst_src != HL_PRSTN_RST)) {
 		prstn = regmap_readl(D0_PSOC_GLOBAL_CONF_REG(mmGLOBAL_CONF_PRSTN));
-		if (prstn) {
-			regmap_writel(0, mmINBAND_TELEMETRY_CONFIG);
+		if (prstn)
 			regmap_writel(0, mmINBAND_TELEMETRY_STATUS);
-		}
 	}
 }
 
